GameData/KerbalRenamer/KerbalRenamer.cfg

```diff
@@ -2,6 +2,754 @@ KERBALRENAMER
+            key = Didia
+            key = Domitia
+            key = Duratena
+            key = Duronia
+            key = Egilia
+            key = Egnatia
+            key = Epidia
+            key = Eppia
+            key = Erucia
+            key = Fabia
+            key = Falcidia
+            key = Favonia
+            key = Flaminia
+            key = Flavonia
+            key = Fulcinia
+            key = Furia
+            key = Furnia
+            key = Gabinia
+            key = Galeria
+            key = Gavia
+            key = Geminia
+            key = Gessia
+            key = Grania
+            key = Gratia
+            key = Heia
+            key = Helvia
+            key = Herminia
+            key = Hirtia
+            key = Horatia
+            key = Hosidia
+            key = Iasidia
+            key = Ignia
+            key = Ituria
+            key = Laberia
+            key = Laenia
+            key = Laronia
+            key = Lentidia
+            key = Licinia
+            key = Livia
+            key = Lucretia
+            key = Laia
+            key = Maccia
+            key = Maenia
+            key = Marcia
+            key = Maria
+            key = Milonia
+            key = Modia
+            key = Munatia
+            key = Mutia
+            key = Naevia
+            key = Nautia
+            key = Neria
+            key = Nonia
+            key = Norbana
+            key = Numeria
+            key = Nummia
+            key = Obellia
+            key = Octavia
+            key = Ofilia
+            key = Oppia
+            key = Opsidia
+            key = Orbia
+            key = Orfia
+            key = Ovidia
+            key = Paccia
+            key = Papia
+            key = Petronia
+            key = Pilia
+            key = Pleminia
+            key = Pontilia
+            key = Praecilia
+            key = Priscia
+            key = Quartinia
+            key = Quirinia
+            key = Rabiria
+            key = Rammia
+            key = Reginia
+            key = Romilia
+            key = Rubria
+            key = Ratia
+            key = Sabellia
+            key = Salviena
+            key = Scribonia
+            key = Sempronia
+            key = Sertoria
+            key = Simplicia
+            key = Staberia
+            key = Stenia
+            key = Strabonia
+            key = Tarquinia
+            key = Tatia
+            key = Ticinia
+            key = Titia
+            key = Travia
+            key = Trebonia
+            key = Tullia
+            key = Tutia
+            key = Ulpia
+            key = Umbilia
+            key = Umbria
+            key = Urbania
+            key = Urvinia
+            key = Valeria
+            key = Velia
+            key = Verginia
+            key = Veturia
+            key = Vibia
+            key = Vipsania
+            key = Vitellia
+            key = Volaia
+        }
+    }
+    Culture
+     {
+        name = Denmark
```

Comment:
This seems to contain a Danish list too. Is that intentional?

Reply:
No, must be a mistake